platform/mwm_version.cpp

```diff
@@ -74,4 +74,10 @@ uint32_t ReadVersionTimestamp(ModelReaderPtr const & reader)
   return version.timestamp;
 }
 
+bool IsSingleMwm(int64_t version)
+{
+  // @TODO(bykoianko) Check this version and move if necessary before small mwm release.
```

Comment:
#pragma message "Check this version and move if necessary before small mwm release."